### DISCUSSION TITLE
Feedback in Profile Achievement's

### DISCUSSION THREAD

**Discussion by charisios10** [community/community#77718] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Dear Github,
So i recently noticed the new achievement's that i started to have into my profile and i have to say they are pretty good for flexing and for making programming a litle bit more fun and exiting rather that being pretty boring. So i would like to see a lot of other fuatures like this that they are about making the community of github fun and exiting. Keep doing that good work.

Have a nice day,
Charisios Charisiadis

**Comment by achmadfha** [comment]
How i can get the badge like u have on ur profile??
